frontend/src/components/requisitionDetail/sections/ReimbursementInstructionsSection.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Typography } from "antd";
+import { RequisitionSectionProps } from "../RequisitionDetail";
+
+const { Title, Text } = Typography;
+
+const ReimbursementInstructionsSection: React.FC<RequisitionSectionProps> = (props) => {
+  const { data } = props;
+
+  if (!data.isReimbursement || data.status !== "AWAITING_INFORMATION" || !data.fundingSource?.reimbursementInstructions) {
+    return null;
+  }
+
+  return (
+    <>
+      <Title level={3} style={{ marginTop: "30px" }}>Reimbursement Instructions</Title>
+      <Text>{data.fundingSource.reimbursementInstructions}</Text>
```

Comment:
It would be nice if links in the text were made into links - a library like [this](https://www.npmjs.com/package/react-linkify) might make that easy or you could just support markdown (but I sorta feel like that's overkill and we don't want to store legit documentation in Piranha - it'd probably just be a link to confluence).

Reply:
changed, this is a pretty cool package